django/db/migrations/questioner.py

```diff
@@ -202,6 +207,24 @@ def ask_merge(self, app_label):
             False,
         )
 
+    def ask_auto_now_add_addition(self, field_name, model_name):
+        "Adding a auto_now_add field to a model"
+        if not self.dry_run:
+            choice = self._choice_input(
+                "You are trying to add field '%s' with 'auto_now_add' attribute to %s without a "
```

Comment:
A few grammar fixes:

`You are trying to add the field '%s' with 'auto_now_add=True' to %s without a default; the database needs to set a default value for existing fields.`